WPF/01.04_practise/MainWindow.xaml.cs

```diff
@@ -0,0 +1,146 @@
+﻿using _01._04_practise.Model;
+using _01._04_practise.Views;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+
+namespace _01._04_practise
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    /// 
+    public partial class MainWindow : Window
+    {
+        public ObservableCollection<Author> AuthorList { get; set; }
+
+        public MainWindow()
+        {
+            InitializeComponent();
+            this.AuthorList = new ObservableCollection<Author>();
+            this.AuthorList.Add(new Author
+            {
+                FirstName = "Mark",
+                LastName = "Twain",
+                BirthDate = new DateTime(1835, 11, 30),
+                Country = Country.USA,
+                Language = Model.Language.English,
+                PlaceOfBirth = "Florida, Missouri",
+                IsNew = false,
+                BooksList = new ObservableCollection<Book>()
+            });
+            this.AuthorList[0].BooksList.Add(new Book { Title = "Tom Soyer", Date = new DateTime(2019, 05, 21),
+                Cost = 52.62m, IsNew = false, Language = Model.Language.English, IsRead = true });
+
+            this.AuthorLV.DataContext = this.AuthorList;
+        }
+
+        private void NewCommand_Executed(object sender, ExecutedRoutedEventArgs e)
+        {
+            FrameworkElement fe = e.Source as FrameworkElement;
+            if (fe.Name == "NewAuthorButton" || fe.Name == "AuthorLV")
+            {
+                Author author = new Author();
+                var authorWindow = new AuthorWindow() { DataContext = author };
+                authorWindow.Owner = this;
+                authorWindow.ShowDialog();
+                if (!authorWindow.DialogResult.Value)
+                    return;
+                author.IsNew = false;
```

Comment:
There is an appropriate method Save() on the EntityBase. Please use that one in all places.